fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/AdminSchemaGenerator.java

```diff
@@ -106,7 +106,10 @@ public void buildSchema(PhysicalDataModel model) {
         this.allTablesComplete.addDependencies(procedureDependencies);
         model.addObject(allTablesComplete);
 
-        model.addProcedure(this.adminSchemaName, SET_TENANT, FhirSchemaConstants.INITIAL_VERSION, () -> SchemaGeneratorUtil.readTemplate(adminSchemaName, adminSchemaName, SET_TENANT.toLowerCase() + ".sql", null), Arrays.asList(allTablesComplete), procedurePrivileges);
+        model.addProcedure(this.adminSchemaName, SET_TENANT, 2,
+                () -> SchemaGeneratorUtil.readTemplate(adminSchemaName, adminSchemaName,
+                        SET_TENANT.toLowerCase() + ".sql", null),
+                Arrays.asList(allTablesComplete), procedurePrivileges);
```

Comment:
this worked ok to automatically update the stored procedure as part of `--update-schema`, right?

does it work the same as calling `--update-procs`?

Reply:
ack it's in two spots.

Reply:
Yes, it's OK. it also happens to be in two spots.

Reply:
newest commit adds the second spot